src/main/java/org/jabref/gui/search/GlobalSearchBar.java

```diff
@@ -329,8 +329,7 @@ private void updateResults(int matched, TextFlow description, boolean grammarBas
 
     private void setHintTooltip(TextFlow description) {
         if (preferencesService.getGeneralPreferences().shouldShowAdvancedHints()) {
-            String genericDescription = Localization.lang("Hint: To search specific fields only, enter for example:<p><tt>author=smith and title=electrical</tt>");
-            genericDescription = genericDescription.replace("<p>", "\n");
+            String genericDescription = Localization.lang("Hint: To search specific fields only, enter for example:\n<tt>author=smith and title=electrical</tt>");
             List<Text> genericDescriptionTexts = TooltipTextUtil.formatToTexts(genericDescription, new TooltipTextUtil.TextReplacement("<tt>author=smith and title=electrical</tt>", "author=smith and title=electrical", TooltipTextUtil.TextType.MONOSPACED));
```

Comment:
@koppor  Here the `<tt>` gets replaced and formatted as monospaced. This is done using a TextFlow Element for that specific line.
I think it could make sense to add this to the general case handling for the other allowed html tags.